examples/tutorials/Going_Deeper_on_Molecular_Featurizations.ipynb

```diff
@@ -451,7 +518,11 @@
   },
   {
    "cell_type": "code",
+<<<<<<< HEAD
```

Comment:
there are some rebase erorrs

I will suggest, it will be easier to make a new pr